daemon/logger/splunk/splunk.go

```diff
@@ -228,12 +231,23 @@ func New(info logger.Info) (logger.Logger, error) {
 		}
 	}
 
+	timeout := defaultHTTPTimeout
```

Comment:
Can we have no default timeout? Making no timeout to 1s timeout is a big change that may not even work for a lot of people.

Reply:
Ah, sorry for the double post; looks like we were both writing a comment during the review meeting 😂

Reply:
yep ok with that, will change to that behavior. Thanks for review.